hvantk/tests/annotation/test_compose.py

```
@@ -0,0 +1,159 @@
+    )
+
+
+@pytest.mark.hail
+def test_compose_left_joins_all_axes_onto_the_spine(hail_session):
+    from hvantk.algorithms.annotation.compose import compose
+
+    prepared_by_axis = {
+        "constraint": _constraint_axis_ht(),
+        "gevir": _gevir_axis_ht(),
+    }
+    composed, manifest = compose(_spine(), prepared_by_axis, _two_axis_spec())
```

Ruff (0.15.21)

[warning] Unpacked variable `manifest` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)